study.md

```diff
@@ -135,15 +135,15 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a breif description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+Hypertext Transfer Protocol is a way of communicating with a server. For example as a client you can request information from a server by typing in a URL which then the server finds information on that URL and returns back to the client which processes the information and gives you a webpage to look at.
 ```
 
 ## Describe what a client is and what a server is
 
  What is a client is and what is a server? How do they interact with each other?
 
 ```md
-<!-- your answer here -->
+A client is a browser that request information from a server. A server is a computer program or a device that provides functionality for other programs or devices, called clients.
 ```
```

Comment:
It's not necessarily always a browser,but that is most common, and it's always making requests to the server.